src/rp2_common/pico_bootrom/bootrom.c

```diff
@@ -7,6 +7,9 @@
 #include "pico/bootrom.h"
 #include "boot/picoboot.h"
 #include "boot/picobin.h"
+#if !PICO_RP2040
```

Comment:
strictly more idiomatic to use `#if HAS_RP2040_RTC`

Reply:
This is the RCP not the RTC - given the functions in this file are also guarded by `!PICO_RP2040` I think this if fine to use here too?